src/extension/gdal/src/main/java/org/geoserver/gdal/GDALStatus.java

```diff
@@ -68,9 +68,13 @@ public boolean isEnabled() {
         return Optional.ofNullable("");
     }
 
-    public String getGDALWrapperJarVersion() {
+    public Optional<String> getGDALWrapperJarVersion() {
         if (isAvailable()) {
-            return GeoTools.getVersion(gdal.class).toString();
+            Version v = GeoTools.getVersion(gdal.class);
+            if (v == null) {
+                return Optional.empty();
```

Comment:
This occurred in "normal" (not dev - running from built jars) environment with a custom build of GDAL.
Sadly, the GeoTools.getVersion call is not especially smart, and just inspects the jar filename for a version number (otherwise returning null).